zulipterminal/ui_tools/buttons.py

```diff
@@ -242,8 +242,8 @@ def __init__(self, stream_id: int, topic: str,
                          width=width,
                          count=count)
 
-        topic_pair = [self.stream_name, self.topic_name]
-        if topic_pair in controller.model.muted_topics:
```

Comment:
I'll emphasize that we should add a leading underscore to `Model.muted_topics` - it'll mark it as 'private' (by convention at least), and any hidden uses of it are made evident too :)